File: snippets/node/update_a_subaccount_s_quota.js
const mailchimp = require("mailchimp-transactional")("YOUR_API_KEY");

async function run() {
  const response = await mailchimp.subaccounts.update({
    id: "UNIQUE_ACCOUNT_ID",
    custom_quota: 100
  });
  console.log(
    `Success! This account's new sending quota is ${response.custom_quota}.`
  );
}

run();

File: snippets/python/update_a_subaccount_s_quota.py
import mailchimp_transactional as MailchimpTransactional
from mailchimp_transactional.api_client import ApiClientError

mailchimp = MailchimpTransactional.Client('YOUR_API_KEY')

def run():
  try:
    response = mailchimp.subaccounts.update({
      "id": "UNIQUE_ACCOUNT_ID",
      "custom_quota": 100
    })
    print("Success! This account's new sending quota is {} ".format(response['custom_quota']))
  except ApiClientError as error:
    print('An exception occurred: {}'.format(error.text))

run()